pkg/oc/cli/admin/release/extract.go

```diff
@@ -66,7 +90,39 @@ type ExtractOptions struct {
 	ImageMetadataCallback func(m *extract.Mapping, dgst digest.Digest, config *docker10.DockerImageConfig)
 }
 
-func (o *ExtractOptions) Complete(cmd *cobra.Command, args []string) error {
+func (o *ExtractOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, args []string) error {
+	switch {
+	case len(args) == 0 && len(o.From) == 0:
+		cfg, err := f.ToRESTConfig()
+		if err != nil {
+			return fmt.Errorf("info expects one argument, or a connection to a 4.0 OpenShift server: %v", err)
+		}
+		client, err := configv1client.NewForConfig(cfg)
+		if err != nil {
+			return fmt.Errorf("info expects one argument, or a connection to a 4.0 OpenShift server: %v", err)
+		}
+		cv, err := client.Config().ClusterVersions().Get("version", metav1.GetOptions{})
+		if err != nil {
+			if apierrors.IsNotFound(err) {
+				return fmt.Errorf("you must be connected to a v4.0 OpenShift server to fetch the current version")
+			}
+			return fmt.Errorf("info expects one argument, or a connection to a 4.0 OpenShift server: %v", err)
```

Comment:
Be consistent with `4.0` vs `v4.0`, also maybe worth stating `v4.x` cluster.

Reply:
Yeah, also needs to be that way in info.  This message will eventually be removed anyway, but it will stick around while people still have 3.11 clusters.